Civi/Core/DAO/Event/PostUpdate.php

```diff
@@ -22,11 +22,18 @@ class PostUpdate extends \Symfony\Component\EventDispatcher\Event {
    */
   public $object;
 
+  /**
+   * @var mixed
+   */
+  public $result;
+
   /**
    * @param $object
+   * @param $result
```

Comment:
We should add data types to the code blocks (I know it's not there for '$object' :-(